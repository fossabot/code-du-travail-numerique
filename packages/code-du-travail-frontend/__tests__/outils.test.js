import React from "react";
import { render } from "@testing-library/react";
import Tool from "../pages/outils/[slug]";

describe("<Tool />", () => {
  it("should render", () => {
    const { container } = render(<Tool />);
    expect(container).toMatchSnapshot();
  });
});
