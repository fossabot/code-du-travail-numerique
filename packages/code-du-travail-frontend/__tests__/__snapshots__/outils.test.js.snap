// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tool /> should render 1`] = `
.c19 {
  padding: 1.25rem 0;
  color: #434956;
}

.c16 {
  margin-bottom: 1rem;
  padding: 0.625rem 1.25rem;
  border: 1px solid transparent;
  border-radius: 0.25rem;
  color: #0c0c0e;
  background: #fee5ad;
  border-color: #fdd57b;
}

.c16 > *:first-child {
  margin-top: 0;
}

.c16 > *:last-child {
  margin-bottom: 0;
}

.c13 {
  display: inline-block;
  padding: 0.625rem 1.5rem;
  font-weight: 600;
  font-size: 1rem;
  line-height: inherit;
  text-align: center;
  vertical-align: middle;
  border: 1px solid;
  border-radius: 1.5rem;
  cursor: pointer;
  -webkit-transition: background-color 250ms ease;
  transition: background-color 250ms ease;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  padding: 1rem;
  color: #434956;
  line-height: 0;
  background-color: transparent;
  border: none;
}

.c13:hover {
  color: #8c94a6;
}

.c22 {
  display: inline-block;
  padding: 0.625rem 1.5rem;
  font-weight: 600;
  font-size: 1rem;
  line-height: inherit;
  text-align: center;
  vertical-align: middle;
  border: 1px solid;
  border-radius: 1.5rem;
  cursor: pointer;
  -webkit-transition: background-color 250ms ease;
  transition: background-color 250ms ease;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  color: #fff;
  background: #005994;
  border-color: #005994;
}

.c22:not([disabled]):hover,
.c22:not([disabled]):active,
.c22:not([disabled]):focus {
  color: #fff;
  background: #0068ae;
}

.c22[disabled] {
  color: rgba(255,255,255,0.4);
  cursor: not-allowed;
}

.c27 {
  margin: 0;
  padding: 0;
  list-style-type: none;
}

.c10:not(:focus):not(:active) {
  position: absolute;
  width: 1px;
  height: 1px;
  margin: -1px;
  padding: 0;
  overflow: hidden;
  border: 0;
  -webkit-clip: rect(0,0,0,0);
  clip: rect(0,0,0,0);
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  text-align: left;
}

.c12 ul {
  z-index: 100;
}

.c12 li[role="option"]:nth-child(2n + 1) {
  background: #f9f9fc;
}

.c8 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0 auto;
  padding: 0;
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  height: 50px;
  margin: 0;
  padding: 0 calc(50px + 0.625rem) 0 1rem;
  font-size: inherit;
  font-family: inherit;
  background: #f9f9fc;
  border: 1px solid #c9d3df;
  border-radius: 1.5rem;
  -webkit-transition: border 250ms ease;
  transition: border 250ms ease;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
}

.c11:focus {
  border-color: #006be6;
  outline: none;
}

.c14 {
  position: absolute;
  top: 1px;
  right: 1px;
  width: calc(50px - 2px);
  height: calc(50px - 2px);
  border-top-right-radius: 0.25rem;
  border-bottom-right-radius: 0.25rem;
}

.c0 {
  background: #fff;
}

.c1 {
  max-width: 1200px;
  margin: 0 auto;
  padding: 0 1.25rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 1.25rem;
}

.c1 > *:first-child {
  margin-top: 0;
}

.c1 > *:last-child {
  margin-bottom: 0;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c4 {
  width: 5rem;
  height: 100%;
  margin-right: 1rem;
}

.c5 {
  font-weight: bold;
  font-size: 1.125rem;
  line-height: 1.2;
}

.c6 {
  display: inline-block;
  padding: 0.5rem 0.625rem;
  font-weight: bold;
  font-size: 0.875rem;
  line-height: 1rem;
  white-space: nowrap;
  text-align: center;
  border-radius: 0.25rem;
  color: #0c0c0e;
  background: #e5f1fe;
  margin-left: 1.25rem;
}

.c7 {
  width: 350px;
}

.c20 {
  max-width: 1200px;
  margin: 0 auto;
  padding: 0 1.25rem;
  text-align: center;
}

.c20 > *:first-child {
  margin-top: 0;
}

.c20 > *:last-child {
  margin-bottom: 0;
}

.c21 {
  max-width: 1200px;
  margin: 0 auto;
  padding: 0 1.25rem;
  max-width: 46.25rem;
  padding: 0;
  margin-bottom: 1.25rem;
  text-align: left;
}

.c21 > *:first-child {
  margin-top: 0;
}

.c21 > *:last-child {
  margin-bottom: 0;
}

.c23 {
  max-width: 1200px;
  margin: 0 auto;
  padding: 0 1.25rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c23 > *:first-child {
  margin-top: 0;
}

.c23 > *:last-child {
  margin-bottom: 0;
}

.c25 + .c24 {
  padding-left: 1rem;
}

.c26 {
  font-weight: bold;
  font-size: 1rem;
}

.c28 {
  margin: 0.5rem 0;
}

.c28 a,
.c28 a:visited {
  color: #434956;
}

.c15 {
  max-width: 1200px;
  margin: 0 auto;
  padding: 0 1.25rem;
  margin: 20%;
  font-size: 1.875rem;
  text-align: center;
}

.c15 > *:first-child {
  margin-top: 0;
}

.c15 > *:last-child {
  margin-bottom: 0;
}

.c17 {
  margin-top: 50px;
  padding: 0.625rem;
  text-align: center;
  background: #f9f9fc;
}

@media (max-width:600px) {
  .c8 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    height: auto;
  }
}

@media (max-width:600px) {
  .c11 {
    padding: 0 1rem;
  }
}

@media print {
  .c1 {
    max-width: 100%;
    padding: 0;
  }
}

@media (max-width:980px) {
  .c1 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media (max-width:600px) {
  .c1 {
    -webkit-align-items: flex-start;
    -webkit-box-align: flex-start;
    -ms-flex-align: flex-start;
    align-items: flex-start;
  }
}

@media (max-width:980px) {
  .c7 {
    width: 70%;
    margin-top: 1.25rem;
  }
}

@media (max-width:600px) {
  .c7 {
    width: 100%;
    margin-top: 1.25rem;
  }
}

@media print {
  .c18 {
    display: none;
  }
}

@media print {
  .c20 {
    max-width: 100%;
    padding: 0;
  }
}

@media print {
  .c21 {
    max-width: 100%;
    padding: 0;
  }
}

@media print {
  .c23 {
    max-width: 100%;
    padding: 0;
  }
}

@media (max-width:600px) {
  .c23 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media (max-width:600px) {
  .c25 + .c24 {
    padding-left: 0;
  }
}

@media print {
  .c15 {
    max-width: 100%;
    padding: 0;
  }
}

<div>
  <header
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <a
          class="c3"
          href="/"
          title="Code du travail numérique - retour à l'accueil"
        >
          <img
            alt="symbole de la Marianne, site officiel du gouvernement"
            class="c4"
            src="/static/assets/img/marianne.svg"
          />
          <span
            class="c5"
          >
            Code du travail
            <br />
            numérique
          </span>
        </a>
        <div
          class="c6"
        >
          Version Bêta
        </div>
      </div>
      <div
        class="c7"
      >
        <form
          class="c8"
        >
          <label
            class="c9"
            for="main-search-input"
          >
            <span
              class="c10"
            >
              Rechercher
            </span>
            <div
              aria-expanded="false"
              aria-haspopup="listbox"
              aria-owns="react-autowhatever-cdtn-documents-suggester"
              role="combobox"
              style="flex: 1;"
            >
              <input
                aria-autocomplete="list"
                aria-controls="react-autowhatever-cdtn-documents-suggester"
                autocomplete="off"
                class="c11"
                id="main-search-input"
                name="query"
                placeholder="Recherche"
                type="search"
                value=""
              />
              <div
                class="c12"
                id="react-autowhatever-cdtn-documents-suggester"
                role="listbox"
              />
            </div>
          </label>
          <button
            class="c13 c14"
            type="submit"
          >
            <svg
              viewBox="0 0 20 20"
            >
              <title />
              <path
                d="M17.876 8.939c0 2.197-.819 4.208-2.123 5.773l4.023 4.022a.72.72 0 0 1 0 1.043.735.735 0 0 1-.522.223.736.736 0 0 1-.52-.223l-4.023-4.023a8.813 8.813 0 0 1-5.772 2.123C4.022 17.877 0 13.855 0 8.939 0 4.022 4.023 0 8.939 0c4.915 0 8.937 4.023 8.937 8.939zM9.001 16.5c4.124 0 7.499-3.375 7.499-7.5S13.125 1.5 9 1.5C4.877 1.5 1.5 4.875 1.5 9s3.376 7.5 7.5 7.5z"
                fill="currentColor"
                fill-rule="evenodd"
              />
            </svg>
          </button>
        </form>
      </div>
    </div>
  </header>
  <main
    id="main"
  >
    <div
      class="c15"
    >
      <div
        class="c16"
      >
        Cet outil est introuvable
      </div>
    </div>
    <div
      class="c17"
    >
      -
    </div>
  </main>
  <footer
    class="c18"
  >
    <div
      class="c19"
    >
      <div
        class="c20"
      >
        <h2>
          Besoin d’un accompagnement personnalisé ?
        </h2>
        <div
          class="c21"
        >
          Les services de renseignement en droit du travail peuvent vous donner des informations juridiques générales relatives au Code du travail, aux conventions collectives, à la jurisprudence. Ils peuvent également vous conseiller et vous orienter dans vos démarches.
        </div>
        <button
          class="c22"
        >
          Contacter les services de renseignement
        </button>
      </div>
    </div>
    <div
      class="c19"
    >
      <div
        class="c23"
      >
        <div
          class="c24 c25"
        >
          <h3
            class="c26"
          >
            Code du travail numérique
          </h3>
          <ul
            class="c27"
          >
            <li
              class="c28"
            >
              <a
                href="/droit-du-travail"
              >
                Le droit du travail
              </a>
            </li>
            <li
              class="c28"
            >
              <a
                href="/glossaire"
              >
                Glossaire
              </a>
            </li>
            <li
              class="c28"
            >
              <a
                href="/a-propos"
              >
                À propos
              </a>
            </li>
            <li
              class="c28"
            >
              <a
                href="/mentions-legales"
              >
                Mentions légales
              </a>
            </li>
            <li
              class="c28"
            >
              <a
                href="mailto:codedutravailnumerique@travail.gouv.fr"
              >
                Contact
              </a>
            </li>
          </ul>
        </div>
        <div
          class="c24 c25"
        >
          <h3
            class="c26"
          >
            Aidez-nous à améliorer cet outil
          </h3>
          <ul
            class="c27"
          >
            <li
              class="c28"
            >
              <a
                href="https://github.com/SocialGouv/code-du-travail-numerique/tree/vX.Y.Z"
                rel="noopener noreferrer"
                target="_blank"
              >
                Contribuer sur Github
              </a>
            </li>
            <li
              class="c28"
            >
              <a
                href="https://github.com/SocialGouv/code-du-travail-numerique/releases/tag/vX.Y.Z"
                rel="noopener noreferrer"
                target="_blank"
              >
                Journal des modifications
              </a>
            </li>
          </ul>
        </div>
        <div
          class="c24 c25"
        >
          <h3
            class="c26"
          >
            En collaboration avec
          </h3>
          <ul
            class="c27"
          >
            <li
              class="c28"
            >
              <a
                href="https://travail-emploi.gouv.fr/ministere/organisation/article/dgt-direction-generale-du-travail"
                rel="noopener noreferrer"
                target="_blank"
              >
                La Direction Générale du Travail
              </a>
            </li>
            <li
              class="c28"
            >
              <a
                href="https://incubateur.social.gouv.fr/"
                rel="noopener noreferrer"
                target="_blank"
              >
                L’incubateur des ministères sociaux
              </a>
            </li>
            <li
              class="c28"
            >
              <a
                href="https://beta.gouv.fr/"
                rel="noopener noreferrer"
                target="_blank"
              >
                beta.gouv.fr
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
